drivers/adc/adc_npcx.c

```diff
@@ -37,6 +37,9 @@ LOG_MODULE_REGISTER(adc_npcx, CONFIG_ADC_LOG_LEVEL);
 #define NPCX_ADC_CHN_CONVERSION_MODE	0
 #define NPCX_ADC_SCAN_CONVERSION_MODE	1
 
+/* ADC threshold detector number */
+#define NPCX_ADC_THRESHOLD_COUNT   DT_PROP(DT_NODELABEL(adc0), threshold_count)
```

Comment:
Unused?

Reply:
This is for PR #40904, which adds threshold functionality.

Reply:
I added in #40904 something like this:
`#if DT_NODE_HAS_PROP(DT_NODELABEL(adc0), threshold_count)
#define NPCX_ADC_THRESHOLD_COUNT   DT_PROP(DT_NODELABEL(adc0), threshold_count)
#else
#define NPCX_ADC_THRESHOLD_COUNT   3
#endif`

Reply:
@perezpri please also add threshold-count in yaml & dtsi. Thanks.

Reply:
Done..
Thanks